src/main/java/com/google/api/codegen/transformer/SurfaceNamer.java

```diff
@@ -65,12 +65,16 @@ public String getNotImplementedString(String feature) {
     return "$ NOT IMPLEMENTED: " + feature + " $";
   }
 
-  /** The full path to the source file  */
+  /**
+   * The full path to the source file
+   */
```

Comment:
Please figure out your formatter settings and change them so that changes like this don't happen. Some people have formatter settings that do the exact opposite of this particular change. Thus, the formatting of one-line comments will keep flipping back and forth as people check in code, which is needless noise.